yt/utilities/on_demand_imports.py

```diff
@@ -56,6 +56,15 @@ class netCDF4_imports:
     _name = "netCDF4"
     _Dataset = None
 
+    def __init__(self):
+        try:
+            import netCDF4
+
+            netCDF4.__version__
```

Comment:
ya, so this is confusing... but the `import` is there to ensure due to an hdf5/netCDF4 import order issue. Without this, if hdf5 has been imported previously then you can get weird file lock issues when trying to read using netCDF4. This was causing the travis tests to fail.... 

The `netCDF4.__version__` is only there to avoid flake8 catching an unused import :)

Reply:
In that case just add `# noqa F401` to the import line